### PR TITLE
Add codes for eiger.cscs.ch.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

**Comment by csadorf** [comment]
For reference: https://confluence.cscs.ch/display/KB/Alps+%28Eiger%29+User+Guide

File: eiger.cscs.ch/default [hunk 1]
```diff
@@ -0,0 +1 @@
+normal
```
Comment (yakutovicha) [inline eiger.cscs.ch/default:1]: I would suggest calling it `mr0`, or `multicore-mr0`.

**Review by yakutovicha — COMMENTED** [review]
Thanks, @csadorf, just a little suggestion.

File: eiger.cscs.ch/default (hunk 1 above)
Comment (csadorf) [reply eiger.cscs.ch/default:1]: Would you also change the computer labels then?

Comment (yakutovicha) [reply eiger.cscs.ch/default:1]: yes, I suggest `eiger-mr0`. The thing is, if you would use another project, you would have to re-create the computer anyways.

**Review by yakutovicha — APPROVED** [review]
lgtm!

**Pull request merged**